packages/components/src/components/combobox/combobox.component.ts

```diff
@@ -0,0 +1,967 @@
+        })}
+      >
+        <label
+          id="label"
+          part="form-control-label"
+          class="form-control__label"
+          aria-hidden=${hasLabel ? 'false' : 'true'}
+          @click=${this.handleLabelClick}
+        >
+          <slot name="label">${this.label}</slot>
+        </label>
+
+        <div part="form-control-input" class="form-control-input">
+          <syn-popup
+            class=${classMap({
+              combobox: true,
+              'combobox--bottom': this.placement === 'bottom',
+              'combobox--disabled': this.disabled,
+              'combobox--focused': this.hasFocus,
+              'combobox--large': this.size === 'large',
+              'combobox--medium': this.size === 'medium',
+              'combobox--open': this.open,
+              'combobox--placeholder-visible': isPlaceholderVisible,
+              'combobox--small': this.size === 'small',
+              'combobox--standard': true,
+              'combobox--top': this.placement === 'top',
+            })}
+            placement=${this.placement}
+            strategy=${this.hoist ? 'fixed' : 'absolute'}
+            flip
+            shift
+            sync="width"
+            auto-size="vertical"
+            auto-size-padding="10"
+          >
+            <div
+              part="combobox"
+              class="combobox__inputs"
+              slot="anchor"
+              @keydown=${this.handleComboboxKeyDown}
+              @mousedown=${this.handleComboboxMouseDown}
+            >
+              <slot part="prefix" name="prefix" class="combobox__prefix"></slot>
+
+              <input
+                part="display-input"
+                class="combobox__display-input"
+                type="text"
+                placeholder=${this.placeholder}
+                .disabled=${this.disabled}
+                .value=${this.displayLabel}
+                autocomplete="off"
+                spellcheck="false"
+                autocapitalize="off"
+                aria-controls="listbox"
+                aria-expanded=${this.open ? 'true' : 'false'}
+                aria-haspopup="listbox"
+                aria-labelledby="label"
+                aria-disabled=${this.disabled ? 'true' : 'false'}
+                aria-describedby="help-text"
+                role="combobox"
+                tabindex="0"
+                @focus=${this.handleFocus}
+                @blur=${this.handleBlur}
+
+                aria-autocomplete="list"
+                aria-owns="listbox"
+                @input=${this.handleInput}
+                @change=${this.handleChange}
+              />
+
+              <input
+                class="combobox__value-input"
+                type="text"
+                ?disabled=${this.disabled}
+                ?required=${this.required}
+                .value=${this.value}
+                tabindex="-1"
+                aria-hidden="true"
+                @focus=${() => this.focus()}
```

Comment:
I think this could be written without an arrow function for better performance, e.g.
```suggestion
                @focus=${this.focus}
```

Reply:
Ah I see, this is fine for me then :)